### ISSUE TITLE
Questions about reproduce of the results

### ISSUE THREAD

**Issue by idan-tankel** [echonet/echo_CLIP#1]
## Description
We came into a problem when trying to reproduce the results on EchoNet-Dynamic dataset.
While predicting results for the `exmaple_video.avi` are easy to do (reproduce example went well) the overall results for the whole dataset are far from the results reported in the paper (~13.2 MAE).

Furthermore, while trying to reproduce the scatterplot, we got data which is seems to be centered around the mean
### Screenshots

![image](https://github.com/echonet/echo_CLIP/assets/50909823/f9ddb416-9141-4ad6-9c34-028a6af47c40)

### To Reproduce

```python
model, _, preprocess_val = create_model_and_transforms(
    "mkaichristensen/echo-clip-r", cache_dir="~/.cache/huggingface/hub",pretrained="...open_clip_pytorch_model.bin",precision="bf16", device="cuda"
)
# wrapping code for the model etc.
...
def test_step(self, batch, batch_idx):
        # this is the test loop
        x = batch["video"].to(torch.bfloat16)
        y = batch[self.gt_fieldname]
        bs,c,T,h,w = x.shape
        x = x.reshape(-1,c,h,w)
        x = F.normalize(self.model.encode_image(x),dim=-1)
        # now the c is not here - the embedding dimension is 512
        x = x.reshape(bs,T,-1)
        x_hat,*_ = compute_regression_metric(
                video_embeddings=x,
                prompt_embeddings=self.query_embeddings,
                prompt_values=self.prompt_values
            )
        
        # now the logic how to convert the similarity into prediction and than loss
        # check if the model is correct. what is the loss here?
        self.log("test_xhat",x_hat)
```

where the `compute_regression_matrix` is the function that is supplied in that repository.
In order to plot - 
```python
list_of_batchs,list_of_xy_pairs,list_of_minmaxs = zip(*results)
mat = torch.stack(list_of_xy_pairs)
offsets = torch.stack(list_of_minmaxs,dim=1)
summary_writer.add_pr_curve(labels=mat[:,0],predictions=mat[:,1],tag="foo")
summary_writer.add_embedding(
    mat=mat
)


# plot errors


# plt.style.use('_mpl-gallery')

# make data:
x = mat[:,0]
y = mat[:,1]
yerr = offsets.detach().cpu().numpy()

fig,ax1 = plt.subplots()
# plot:
plt.figure()
plt.errorbar(x, y, yerr, fmt='o',ecolor="cornflowerblue",capthick=1,lw=2)
# for EF the scale is 1 to 100 percents of efficiency
ax1.set(xlim=(0, 100), xticks=np.arange(0, 100,10),
       ylim=(0, 100), yticks=np.arange(0, 100,10))
ax1.set_xlabel("real EF")
ax1.set_title("EF plot")
plt.axline((0, 0), (100, 100), color='gray',linestyle="--",linewidth=0.9)
ax1.plot([0, 0], [100, 100], transform=ax1.transAxes)
plt.show()
plt.savefig("docs/assets/img/plot")
```

## Notes
 - [ ] The echoNet Dynamic dataset version we have consist 7465 train examples, 1288 validation examples, 1277 test examples

**Comment by echonet** [comment]
Thanks for your feedback. To clarify, EchoCLIP requires data input of size 224x224 rather than 112x112. We have access to data was processed to the appropriate size. The public EchoNet-Dynamic dataset is 112x112 and we are working to have a higher resolution release in the future. To test EchoCLIP now, you can try resampling to a higher rez or trying another dataset.

**Comment by mkaic** [comment]
I would also recommend using EchoCLIP weights rather than EchoCLIP-R weights. EchoCLIP-R was the weaker of the two models we proposed in the paper when it came to zero-shot EF prediction.

**Comment by duyao-art** [comment]
I implement the similar zeros-shot LVEF prediction on Echo-Dynamic Test Set with resolution of 112X112, and the MAE is about 9. When I use F.interpolate to upsampling to 224X224, the result seems worse, with an MAE of about 12. 
So, I wonder if this model is not suitable for downstream task on the public released Echo-Dynamic Dataset like LEVF prediction. And when will you release the higher res Echo dataset? @idan-tankel @echonet 

I would appreciate it if you could reply in your free time.
